vpdq/benchmark/benchmark.py

```diff
@@ -123,17 +111,15 @@ def run_python(
 
 
 def run_cpp(
-    ffmpegPath: str,
     secondsPerHash: int,
     downsampleFrameDimension: int,
 ):
+    Path.mkdir(OUTPUT, exist_ok=True)
```

Comment:
Need this directory or the hashes won't be output. This avoids having to make it manually.